helpers/signatureVerification.ts

```diff
@@ -0,0 +1,121 @@
+/* 
+Helper to check if signature is a single signature
+*/
+export function isSignatureContract(signature_script){
+    const PUSHDATA1   = 12, //0x0C
+          PUSHNULL    = 11, //0x0B,
+          PUSHSYSCALL = 65,  //0x41
+          SCRIPTLEN   = 41
+   
+    const script = Buffer.from(signature_script, "hex");
+    if(script.length != SCRIPTLEN || script[0] != PUSHDATA1 || script[1] != 33 || script[35] != PUSHNULL || script[36] != PUSHSYSCALL || script.readUInt32LE(37) != 2014135445){
+        return false;
+    }    
+    return true;
+}
+
+/* 
+Helper to check if signatures are multi-sig
+*/
+
+interface SignParamsLike {
+    publicKeyCount: number;
+    signatureCount: number;
+ }

```

Comment:
this can be removed based on https://github.com/CityOfZion/neon-js/issues/591#issuecomment-707607923